src/fides/data/sample_project/cookie_house/src/pages/index.tsx

```diff
@@ -25,10 +31,22 @@ const IndexPage = ({
         <link rel="icon" href="/favicon.ico" />
         <link rel="stylesheet" href="https://rsms.me/inter/inter.css"></link>
       </Head>
-
+      {gtm_key && (
+        <Script id="google-tag-manager" strategy="afterInteractive">
+          {`
+            (function (w, d, s, l, i) {
+              w[l] = w[l] || []; w[l].push({
+                'gtm.start':
+                  new Date().getTime(), event: 'gtm.js'
+              }); var f = d.getElementsByTagName(s)[0],
+                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
+                  'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
+            })(window, document, 'script', 'dataLayer', '${gtm_key}');
+          `}
+        </Script>)}
```

Comment:
nitpick: I'd suggest using a ternary here (e.g. `gtm_key ? <JSX if truthy> : <JSX otherwise>`), like this:
```suggestion
      {gtm_key ? (
        <Script id="google-tag-manager" strategy="afterInteractive">
          {`
            (function (w, d, s, l, i) {
              w[l] = w[l] || []; w[l].push({
                'gtm.start':
                  new Date().getTime(), event: 'gtm.js'
              }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                  'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', '${gtm_key}');
          `}
        </Script>) : null }
```

In this specific example it's not a problem, because the result of `gtm_key && <JSX>` isn't going to surprise you anytime. When `gtm_key` is empty/undefined, that'll return undefined and it won't get rendered, since React doesn't render most falsey values!

The gotcha to know about would be if `gtm_key` was set to `gtm_key = 0`. In that case, you'd render a random `0` to the page. Always using an explicit ternary avoids that gotcha 👍. See the "Pitfall" callout here: https://react.dev/learn/conditional-rendering#logical-and-operator-